pylintrc

```diff
@@ -209,7 +209,7 @@ max-attributes=7
 min-public-methods=2
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=20
+max-public-methods=22
```

Comment:
Why up the limit here instead of adding a `# pylint: disable=...` directive in the relevant file?